spec.emu

```diff
@@ -97,13 +97,13 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
           </tr>
           <tr>
             <td>
-              <ins>[[ModuleReflection]]</ins>
+              <ins>[[SourcePhase]]</ins>
```

Comment:
This should probably just be `[[Phase]]`? Or at least, not contain `Source` in the name.